frappe/email/receive.py

```diff
@@ -3,6 +3,7 @@
 
 import datetime
 import email
+import email.charset
```

Comment:
Do we still need this?

Reply:
Hmm, linter didn't point it out for some reason. Removed.